add-template-suffix.ts

```diff
@@ -10,7 +10,7 @@ const suffix = '.template';
 
 console.log(`Renaming files in ${path}...`);
 const files = getRecursiveFileList(path).filter(
-  (file) => !file.endsWith(suffix)
+  (file) => !file.endsWith(suffix) && !file.endsWith('.DS_Store')
 );
```

Comment:
We can always ignore the `.DS_Store` files so we filter them here and I removed them from the template.